src/main/scala/it/pagopa/interop/backendforfrontend/api/impl/SupportApiServiceImpl.scala

```diff
@@ -211,7 +216,7 @@ final case class SupportApiServiceImpl(
         id = selfcareId,
         name = tenant.name,
         roles = Seq(Role(partyRole = SELFCARE_OPERATOR_ROLE, role = SUPPORT_ROLE))
-      ).toJson.toString,
+      ).toJson.asJsObject.toString,
```

Comment:
I didn't try, but I think this may work
```suggestion
      ).toJson.asJsObject,
```